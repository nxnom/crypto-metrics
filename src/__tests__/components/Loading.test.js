import { render } from '@testing-library/react';

import Loading from 'src/components/Loading';

describe('Loading', () => {
  it('renders correctly', () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
