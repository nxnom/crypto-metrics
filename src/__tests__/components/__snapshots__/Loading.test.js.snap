// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="loading"
    >
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
